Source/Diagnostics/FlushFormats/FlushFormatAscent.cpp

```diff
@@ -12,7 +12,7 @@ FlushFormatAscent::WriteToFile (
     amrex::Vector<amrex::Geometry>& geom,
     const amrex::Vector<int> iteration, const double time,
     const amrex::Vector<ParticleDiag>& particle_diags, int nlev,
-    const std::string prefix, bool plot_raw_fields,
+    const std::string prefix, int /*file_min_digits*/, bool plot_raw_fields,
```

Comment:
I'm not sure I understand why this and some of the later constants are commented out. Is it some kind of varying declaration?